DNN Platform/Library/Entities/Urls/RewriterUtils.cs

```diff
@@ -35,7 +35,7 @@ namespace DotNetNuke.Entities.Urls
 {
     public class RewriterUtils
     {
-        internal static readonly Regex ServicesFrameworkRegex = new Regex("DesktopModules/.+/API/", RegexOptions.Compiled);
+        internal static readonly Regex ServicesFrameworkRegex = new Regex("desktopModules/.+/api/|api/.+/action/", RegexOptions.Compiled);
```

Comment:
That's not true if you include the forward slashes in the Regex as  DNN-8347 will not be valid in that case.

As for ignoring the case, yes it has a slight overhead but if you are matching without the wild card (`.+`), it won't degrade the performance by any noticeable/measurable value.